common/lib/xmodule/xmodule/mongo_utils.py

```diff
@@ -30,10 +30,18 @@ def connect_to_mongodb(
     handles AutoReconnect errors by retrying read operations, since these exceptions
     typically indicate a temporary step-down condition for MongoDB.
     """
+    # Appsembler Specific: This code allow us to have a default replica set value
+    # in our configuration playbooks.
+    if kwargs.get('replicaSet', '') == '':
+        # If the replicaSet is empty let's set it as None, that is the default
+        # value that pymongo has. If not, will look for a '' replica set.
+        kwargs.get('replicaSet') = None
```

Comment:
```suggestion
        kwargs.get('replicaSet') = None
```

I believe that would be a syntax error.

Reply:
Absolutely, It's fixed now 🤦‍♂ I'm really tired already 😄